domain_utils/domain_utils.py

```diff
@@ -236,3 +236,29 @@ def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True, ex
         loc_out=loc_out,
         path_out=path_out,
     )
+
+def get_scheme(url):
+
+    """
+    Given an url extract from it the scheme
+    
+    Parameters
+    ----------
+    url: string 
+        The URL from where we want to get the scheme
+    
+    Returns
+    ----------
+    string
+        Returns the scheme with a default of 'blank' if no schema is provided
+    """
+
+    BLANK = 'blank'
```

Comment:
Let's pull this out and to the top of the module as a global constant. Then the tests can import `BLANK` and check against that. Which will make it easier to maintain if we want to change the return value.